clang/include/clang/Driver/Options.td

```diff
@@ -8545,6 +8545,13 @@ def dxc_entrypoint : Option<["--", "/", "-"], "E", KIND_JOINED_OR_SEPARATE>,
                      Group<dxc_Group>,
                      Visibility<[DXCOption]>,
                      HelpText<"Entry point name">;
+def dxc_hlsl_version : Option<["/", "-"], "HV", KIND_JOINED_OR_SEPARATE>,
+                     Group<dxc_Group>,
+                     Visibility<[DXCOption]>,
+                     HelpText<"HLSL Version">,
+                     Values<"2016, 2017, 2018, 2021, 202x">,
+                     NormalizedValuesScope<"LangStandard">,
+                     NormalizedValues<["hlsl2016", "hlsl2017", "hlsl2018", "hlsl2021", "hlsl202x"]>;
```

Comment:
nit: Do we need these? The `-help` spew should be driven by the `Values` entry right?
```suggestion
```